yaml/sla-error-codes.yaml

```diff
@@ -3,6 +3,7 @@
 
 # Printer code
 # SL1            10xxx
+# M10            42xxx
```

Comment:
shouldn it be 29xxx? as 0X001D is 29 in decimal

Reply:
I would also say 29.

Reply:
fixed